subsys/bluetooth/audio/pbp.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <stdlib.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/pbp.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/check.h>
```

Comment:
Could you clean these up? e.g. `conn` and `gatt` seems unused, but probably several others